pkgs/os-specific/linux/pam/default.nix

```
@@ -26,6 +26,12 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  # Case-insensitivity workaround for https://github.com/linux-pam/linux-pam/issues/569
+  prePatch = lib.optionalString stdenv.buildPlatform.isDarwin ''
+    rm CHANGELOG
+    touch ChangeLog
+  '';
```

```suggestion
  postPatch = if (stdenv.isDarwin && (stdenv.buildPlatform != stdenv.hostPlatform)) then ''
    rm CHANGELOG
    touch ChangeLog
  '' else null;
```

I image there will be a huge rebuild otherwise.

---

Thanks! I assumed it would be fine since `linux-pam` is only supposed to be used on Linux.